.github/workflows/planned_testing_caller.yml

```diff
@@ -19,7 +19,9 @@ jobs:
     uses: ./.github/workflows/planned_testing.yml
     with:
       target_list: '["host_x86_64_linux", "host_aarch64_linux", "host_riscv64_linux", "host_i686_linux", "host_refsi_linux", "host_x86_64_windows" ]'
+      ock: true
```

Comment:
Can't add the comment above but I think it's worth commenting out the pull request as a trigger and just use it for localized testing.

Reply:
OK - will comment out `pull_request:` with an appropriate note to that effect.

Reply:
Done